### PR TITLE
first pass at simple gui

### PR DESCRIPTION
**Pull request by lisazacarias**
Hi all,

I'd like your feedback on this little tutorial I'm building both as people who have recently been onboarded and as people who have a vested interest in spreading the knowledge. Any and all comments and suggestions welcome!

### CODE REVIEW

**Comment by lisazacarias** [comment]
@nneveu @s-aderhold @scottmarino @Derikka any thoughts? I'd like to get this released within the next week or two! And barring any objections I'll just merge my PR in if I don't get comments by then lol

File: pv_readout/pv_readout.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+class PVReadoutGUI(Display):
+    
+    def ui_filename(self):
+        """
+        Your code here
+        """
+        return "example.ui"
+    
+    def __init__(self, parent=None, args=None):
+        super().__init__(parent=parent, args=args, ui_filename="example.ui")
```
Comment (nneveu) [inline pv_readout/pv_readout.py:13]: I'm not a fan of input names being the same as variables, args=args etc. I think that's my personal preference / I get that it's convenient/consice. I've just had a lot of students get confused by this in the past

**Review by nneveu — APPROVED** [review]
I think this is a great starting place and can be merged as is. I have a lot of comments on where details can be added to the instructions, but those can be added mainly to the confluence page. Thanks for putting this together!

File: pv_readout/pv_readout.py (hunk 1 above)
Comment (lisazacarias) [reply pv_readout/pv_readout.py:13]: Hmm that's an interesting point. Personally I take it as a signal that the values are intended to be propagated down through sub inits unchanged (I'm having a hard time phrasing what I mean, sorry if that doesn't make sense)

**Pull request merged**